roles/ember-csi/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+apb_action: "provision"
+ember_csi_namespace: "ember-csi"
+repo_name: "quay.io/kirankt"
```

Comment:
Please use embercsi's repository instead.
embercsi/ember-csi-operator:v0.0.3 is available there.

Reply:
Changed. Thanks.